solvers/bcd_celer.py

```diff
@@ -0,0 +1,342 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from numpy.linalg import norm
+    from numba import njit
+    from mtl_utils.common import sum_squared, _get_dgemm, norm_l21
+
+
+def dual_mtl(alpha, norm_Y2, Theta, Y):
+    n_samples = Y.shape[0]
+    Y /= (n_samples * alpha)
+    d_obj = ((Y - Theta) ** 2).sum()
+    d_obj *= 0.5 * alpha ** 2 * n_samples
+    d_obj += norm_Y2 / (2. * n_samples)
+    return d_obj
+
+
+def primal_mtl(W, alpha, R):
+    n_samples = R.shape[0]
+    p_obj = sum_squared(R) / (2 * n_samples)
+    p_obj += norm_l21(W, 1, copy=True) * alpha
```

Comment:
why copy ? I think this is inherited from MNE but I don't see the use of this param